docs/DeveloperGuide.adoc

```diff
@@ -109,7 +104,7 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 .Interactions Inside the Logic Component for the `delete 1` Command
 image::DeleteSequenceDiagram.png[]
 
-NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+NOTE: The lifeline for `DeleteNoteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
 [[Design-Model]]
 === Model component
```

Comment:
The text in the diagram seems to be too small as well and the diagram appears a bit cluttered. Perhaps the Model class diagram could be split into sub components for Card, Notes etc.
![image](https://user-images.githubusercontent.com/26473264/67543190-c9512d80-f722-11e9-8c18-55ccdda30904.png)